packages/synapse-react-client/src/components/CardContainer/CardContainer.tsx

```diff
@@ -63,8 +65,8 @@ export function CardContainer(props: CardContainerProps) {
   } = props
   const infiniteQueryContext = useInfiniteQueryContext()
   const { NoContentPlaceholder } = useQueryVisualizationContext()
-  const { data, appendNextPageToResults, hasNextPage } = infiniteQueryContext
-
+  const { appendNextPageToResults, hasNextPage } = infiniteQueryContext
+  const data = useAtomValue(tableQueryDataAtom)
```

Comment:
`useAtomValue` is how we can access the value of a state variable.

You'll notice I did not completely remove `QueryContext`. In this pass I just focused on state referenced within SynapseTable and its child components because updates to these components were causing the largest performance hit.